src/activity-functions-function-notation.xml

```diff
@@ -662,83 +578,221 @@
+                             <member>\text{light years}</member>
+                             <member>\text{meters}</member>
+                        </set>
                     </var>
-                    <var name="$numSolutions2">
+                    <pg-code>
+                        $dist= Real(40);
+                        $miles = PopUp(["?", "feet", "miles", "light years", "meters"], "miles");
+                    </pg-code>
+                </setup>
+                <statement>
+                    <p>Remember in a vertical table the inputs are found in the left column (independent variable) while outputs are found in the right column (dependent variable)</p>
+                    <table>
+                        <tabular top="medium" bottom="medium" left="medium" right="medium">
+                            <col halign="center"/>
+                            <col halign="center"/>
+                            <row>
+                                <cell><m>t</m> (hours after 8 a.m.)</cell>
+                                <cell><m>D = f(t)</m> (miles from downtown)</cell>
+                            </row>
+                            <row>
+                                <cell><m>0</m></cell>
+                                <cell><m>10</m></cell>
+                            </row>
+                            <row>
+                                <cell><m>0.5</m></cell>
+                                <cell><m>12</m></cell>
+                            </row>
+                            <row>
+                                <cell><m>1.5</m></cell>
+                                <cell><m>20</m></cell>
+                            </row>
+                            <row>
+                                <cell><m>2.5</m></cell>
+                                <cell><m>40</m></cell>
+                            </row>
+                            <row>
+                                <cell><m>3.0</m></cell>
+                                <cell><m>46</m></cell>
+                            </row>
+                            <row>
+                                <cell><m>3.5</m></cell>
+                                <cell><m>47</m></cell>
+                            </row>
+                        </tabular>
+                    </table>
+                    <p>Use the table to evaluate <m>f(2.5)</m></p>
+                    <p><m>f(2.5)= </m><var name="$dist" width="5" /> <var name="$miles" form="popup" /></p>
+                </statement>
+            </webwork>
+        </exercise>
+
+        <p>In general when the inputs and outputs are nothing in particular, we use <m>x</m> as input and <m>y</m> as output so that <m>y = f(x)</m>.  We use <q>x</q> as symbol to hold a spot for the actual input value while <m>y</m> is a spot holder for whatever the output value will be.  The <m>x</m> basically says, <q>When you get the input, put it here</q> in all the places where the <m>x</m> is.</p>
```

Comment:
Not sure if `<q>x</q>` and `<m>y</m>` should be `<q><m>x</m></q>` and `<q><m>y</m></q>`.